templates/e2e-llm-workflows/README.ipynb

```diff
@@ -805,47 +841,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 10,
    "id": "8e041050",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Locations\n",
     "artifacts_dir = '/mnt/cluster_storage'  # storage accessible by head and worker nodes\n",
-    "model = 'meta-llama/Meta-Llama-3-8B-Instruct'\n",
-    "uuid = 'gokum:iuwom'  # REPLACE with your NAME + MODEL ID (from Job logs)\n",
-    "artifacts_path = (\n",
-    "    f\"{os.environ['ANYSCALE_ARTIFACT_STORAGE'].split(os.environ['ANYSCALE_CLOUD_STORAGE_BUCKET'])[-1][1:]}\"\n",
-    "    f\"/lora_fine_tuning/{model}:{uuid}\")"
+    "model_id = model_info.id \n",
+    "artifacts_path = f\"{os.environ['ANYSCALE_ARTIFACT_STORAGE']}/lora_fine_tuning/{model_id}\""
```

Comment:
i think this is supposed to be `model_info.storage_uri`?

Reply:
Hmm don't think so. Actually `model_info.storage_uri` has the storage URI for the best checkpoint *before* forwarding, so it is a long ray train checkpoint path (like TorchTrainer_2024_8_31_12_17_19/epoch_0/...). \


I thought of using the forwarded checkpoint path directly because it is common for all LoRAs. 

Wdyt?

Reply:
Oh I see, yeah this should be ok for now